test/README.md

```diff
@@ -84,10 +84,70 @@ ssh and web.  See the "Helpful tips" section below.
 
 ## Pixel tests
 
-The verify test suite contains ["pixel tests"](https://cockpit-project.org/blog/pixel-testing.html).
-Make sure to create the test/reference submodule before running tests which contain pixel tests.
+Pixel tests in Cockpit ensure that updates of our dependencies or code changes
+don't break unexpectedly break the UI, for example slight changes of layout,
+padding, color and everything which isn't easily tested by a human. They also
+give us confidence that an update of our UI Framework doesn't introduce changes
+in how Cockpit looks.
+
+Pixel tests are in essence quite simple, they make a screenshot of the current
+state and compare this to a known good state, if there is a difference the test
+fails and a pixel difference is shown. This works as our tests run in the
+[cockpit/tasks container](https://quay.io/repository/cockpit/tasks) which pins
+the browser and font rendering so repeated runs provide the same pixels. To
+generate new pixels this tasks container should be used as your own browser and
+font rendering software might generate different results. For more information
+read the ["introduction blog post"](https://cockpit-project.org/blog/pixel-testing.html).
+
+The test images are stored in git submodule in the `test/reference` directory and be fetched with:
+
+```sh
+./test/common/pixel-tests fetch
```

Comment:
`fetch` does not retrieve things from the Git repo, do you mean `update`?

Reply:
Ah, `pull`!

Reply:
Oh, TIL, pull is deprecated for `update` good to know :)